bindings/matrix-sdk-ffi/src/element.rs

```diff
@@ -0,0 +1,21 @@
+use serde::Deserialize;
+
+use crate::ClientError;
+
+/// Well-known settings specific to ElementCall
+#[derive(Deserialize, uniffi::Record)]
+pub struct ElementCallWellKnown {
+    widget_url: String,
+}
+
+/// Element specific well-known settings
```

Comment:
Do we have some docs where this is defined? Could you please add a pointer to the docs if so.

Reply:
According insider sources, it seems that this well-known file is super secret stuff and no public documentation exists.